src/reducers/__tests__/console.test.jsx

```diff
@@ -25,11 +28,12 @@ test('evaluateConsoleEntry adds entry to history', () => {
 });
 
 test('consoleValueProduced adds value to existing entry', () => {
-  const value = 2;
-  const key = '123';
+  const consoleValueProducedAction =
+    actionFactory[consoleValueProduced].build();
```

Comment:
Obviously this could be structured more normally as ES6 modules that just have a named export for each action factory, but I like this a little better because:

* It makes it very clear that the factory is _not_ an action creator function
* It ties the factory explicitly to the action creator itself